src/app.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const { Cat } = require('./models');
+
+const app = express();
+
+app.use(express.json());
+
+app.post('/cats',async (req, res) => {
+    await Cat.create(req.body)
+    .then((cat) => res.status(201).json(cat))
+    .catch((err) => {
+        console.error('error', err);
+        res.status(400).json({ error: 'Bad Request' });
+    });
+});
+
+app.get('/cats',async (req, res) => {
+    await Cat.findAll({ where: req.query })
+    .then((cat) => res.status(201).json(cat))
```

Comment:
Status code 201 means created, which is at better use for a PUT or PATCH request. 
I would use 200 for GET which means ok.